lua/telescope/builtin/files.lua

```diff
@@ -145,106 +145,126 @@ files.grep_string = function(opts)
   }):find()
 end
 
--- TODO: Maybe just change this to `find`.
--- TODO: Support `find` and maybe let people do other stuff with it as well.
-files.find_files = function(opts)
+local function which_find_command(opts)
+  local command = opts.find_command
+  if command then
+    return command[1]
+  end
+  if 1 == vim.fn.executable "fd" then
+    return "fd"
```

Comment:
With only that we will go from `{ "fd", "--type", "f" }` to only `"fd"` which is something different.

Reply:
I'm going to fix this